packages/client/hmi-client/src/router/index.ts

```diff
@@ -7,6 +7,7 @@ import SimulationPlanPlayground from '@/views/SimulationPlanPlayground.vue';
 import TheiaView from '@/views/theia.vue';
 import DocumentView from '@/views/document.vue';
 import Simulation from '@/views/Simulation.vue';
+import ModelRenderer from '@/views/ModelRenderer.vue';
```

Comment:
For consistency let's just call it Model instead of ModelRenderer

Reply:
Sure thing, changed all occurrences to Model instead